app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -47,6 +47,7 @@ def show
             param :cron_line, String, :required => false, :desc => N_('How often the job should occur, in the cron format')
             param :max_iteration, :number, :required => false, :desc => N_('Repeat a maximum of N times')
             param :end_time, DateTime, :required => false, :desc => N_('Perform no more executions after this time')
+            param :start_at, DateTime, :required => false, :desc => N_('Schedule the first iteration after this time')
```

Comment:
This trips up hammer. The value ends up under both `recurrence` and `scheduling` keys in the request being sent to the server.

```
[DEBUG 2021-07-28T14:34:23 API] Params: {
    "job_invocation" => {
                     "inputs" => {
            "playbook" => "asd"
        },
                        "ssh" => {},
                 "recurrence" => {
            "cron_line" => "0 0 * * *",
             "start_at" => "2021-07-31T17:00:00+00:00"
        },
                 "scheduling" => {
            "start_at" => "2021-07-31T17:00:00+00:00"
        },
        "concurrency_control" => {},
               "search_query" => "name = test-00.example.com",
                    "feature" => "ansible_run_playbook",
             "targeting_type" => "static_query"
    }
}
[DEBUG 2021-07-28T14:34:23 API] Headers: {}
[DEBUG 2021-07-28T14:34:23 API] Using authenticator: HammerCLIForeman::Api::InteractiveBasicAuth
[ERROR 2021-07-28T14:34:24 API] 500 Internal Server Error
[DEBUG 2021-07-28T14:34:24 API] {
    "error" => {
        "message" => "ERF42-4606 [Foreman::Exception]: Cannot specify both recurrence and scheduling"
    }
}
```